### DISCUSSION TITLE
WebUI not reachable after subnet changed

### DISCUSSION THREAD

**Discussion by clemone210** [qdm12/ddns-updater#301] [Q&A]
Hi,
I am not sure if you can help me..
My network gear broke down and after a replacement, I thought I'd give the whole network a rework.
I created 3 different subnets. 1 is new, which is the management subnet. Currently, there are no firewall rules, so everything is reachable through each subnet.

So I switched from:
192.168.5.1/24 
192.168.107.1/24 

to

1.10.100.1/24 
1.10.101.1/24 
1.10.102.1/24

So before the rework, my pi had the address 192.168.5.20, now it got 1.10.102.2
I recreated all my docker containers. All are reachable except yours. I even tried to change ports, but I am not able to reach it.

Do you have any suggestion what can cause it?

**Comment by qdm12** [comment]
The Web UI is (still) a dumb pre-rendered html page being served so it's probably a port misconfiguration? Check that the logs of the container matches what you expect?
